src/components/LowerLayout/GeneralInfo/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import global from '../../../styles.module.css';
+import classes from './style.module.css';
+
+const createList = listContainer => listContainer.map(({ list }) => (
+  <ul key={list}>
+    <li>{list}</li>
+  </ul>
+));
+
+const createAdditionalInfo = additionalInfo => additionalInfo.map(({ info, link }) => (
+  <p key={info}>
+    {info}
+    {link &&
```

Comment:
Perhaps also break this into a function, is a bit hard to read across multiple lines.

Reply:
For example `{link && createLink(link)}`